### DISCUSSION TITLE
A systematic error in the representation of redshift and the modeling of the tolman test?

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#296] [Astrophysics and Physics]
https://www.onlinescientificresearch.com/articles/wrong-definition-and-wrong-implications-of-cosmic-red-shift-correction-and-possible-solutions.pdf

Hi dear @RedshiftDrift I would love if you could analyze this seemingly simple and unorthodox view on cosmological redshift

> Since 1912, cosmologists strongly believe in the currently accepted definition of cosmic red shift as - ratio of increase in wavelength to the laboratory wavelength of photon. We would like to emphasize the point that, this definition is absolutely wrong with reference to photon energy. Clearly speaking, true cosmic red shift must be defined as the ratio of loss in energy of photon to the energy of photon at laboratory. In terms of wavelength, photon red shift must be defined as the ratio of increase in wavelength to the observed wavelength of photon. We have published this definition in our recent publications and sincerely appeal the science community and cosmologists to review the basics of Lambda cosmology, cosmic acceleration and dark energy in an unbiased approach. Regarding Tolman Surface Brightness test, we sincerely appeal the cosmologists to review and reanalyze the observations and calculations as per the corrected cosmic redshift definition with reference to cosmic expansion and cosmic rotation. Here we would like to emphasize the point that, (1+z) should be replaced by (1+znew). Upper limit of (1+znew) is 2 and upper limit of (1+znew)power4 is 16.

> A very simple math formula is misleading modern cosmologists to develop and
reinforce the accelerated model of cosmology. With reference to photon energy, true cosmic red
shift must be defined as the ratio of loss in energy of photon to the energy of photon at laboratory.
In terms of wavelength, photon red shift must be defined as the ratio of increase in wavelength to
the observed wavelength of photon but not the laboratory wavelength. If one is willing to
consider this point, in terms of traditional red shift (z), true cosmic red shift can be expressed as
z_new = [z/(z+1)]. We are afraid, but sure to say that wrong definition and wrong formula of
galactic red shift led cosmologists to wrongly introduce the cosmic acceleration and dark energy
concepts. 

> Correction to the 100 years Old Definition of Cosmic Red Shift

This author reinterpret the redshift mathematical formalism in the first page of the paper. The maths seems reasonable and simple intuitively but I'm no sure of wether he is correct and of the full implications of his finding, especially regarding a reinterpretation of how we must perform/interpret the Tolman test, but this might be groundbreaking.

(there are only a few lines of equations (page 1 and 2) @HanDeBruijn ping since you're the math nerd

I believe those authors have some merits, at the very least, they share my insight (never mentionned in the literature) that the cosmic realtime temperature drift has intrinsic merits and is more feasable than measuring the redshift drift, even though unlike me they don't show how trivially feasable it is to measure via >100 times sensitivity improvement in upcoming cmb distortions balloons, they mention that measuring cosmic temperature as a distance marker allows to bypass many systematics associated with redshift since temperature is the only distance measure independent of light systematics.
> following the rate of decrease
in future cosmic temperature with reference to current cosmic
temperature, true nature of cosmic expansion rate can be decided
> Thus, by
following future cosmic temperature, there is 100% scope for
understanding and deciding the true nature of cosmic expansion
rate independent of galactic distances.

Moreover the hubble tension is it seems empirically proven to be an artefact since there is no temperature-distance tension
as I previously mentionned:
> BTW a unique probe of the hubble tension would be to compare the anomalous coma cluster distance with the CMB temperature redshift of the Coma cluster!! This might be historic and groundbreaking
https://arxiv.org/abs/2409.14546

Maybe this will end up being the best indirect contribution from Riess
btw there is a variant more recent paper https://www.scienceopen.com/hosted-document?doi=10.14293/PR2199.001121.v3

Considering the temperature associated with recombination period and
current cosmic temperature, it seems that
≅ 135 Mpc . 
end of page 6, equations seems extremely insightful, I believe the whole cosmology community has missed extremely important insights by not leveraging temperature as a distance marker

> we would like to emphasize the point that above 160 million solar masses, galactic baryon matter acquires super gravitational behaviour.
https://www.researchgate.net/publication/369204981_Light_Speed_Expanding_Hubble-Hawking_Universe
https://www.researchgate.net/publication/308386482_Is_dark_energy_-_an_alias_of_cosmic_rotational_kinetic_energy
https://www.preprints.org/frontend/manuscript/20b1836b61cdf0aed7b4b5b19ee4d6d2/download_pub
https://iopscience.iop.org/article/10.1088/1475-7516/2012/02/013
https://www.aanda.org/articles/aa/full_html/2013/03/aa20613-12/aa20613-12.html
https://iopscience.iop.org/article/10.1088/0004-637X/709/1/386
https://iopscience.iop.org/article/10.1088/1674-4527/20/7/109
https://arxiv.org/abs/2009.14417
could be explained by deviation from the locally measured value of T0, if the adiabatic scaling of the temperature
with redshift still holds [11, 12].
evertheless,
near future surveys with e.g. the Simons Observatory are expected to detect thousands of clusters; this opens up
the possibility of complementing CMB anisotropy with an independent probe that continuously samples T(z) up to
z ≈ 2. 
Insight gained on the Hubble tension is that treating the CMB temperature as a free parameter considerably lowers
the previous ∼ 5σ tension at a ‘cost’ of a moderately lower temperature, at the 2σ level, than would be expected
if the locally measured T , extrapolated to z ≈ 1100, holds at the FIRAS-deduced precision level. Similarly reduced
level of Hubble tension can also be achieved by a slightly positive value of Ωk, i.e., in a model with geometrically open
spacetime. Without the local Hubble prior all dataset combinations considered in this work favor either a lower T at
last scattering, or the FIRAS value, and either flat or closed universe. 
> Therefore, it is possible that the deduced ‘closed’ space at the
∼ 2σ confidence level actually implies that space is flat, but that T at recombination is ∼ 2σ (i.e. a few percent)
higher than inferred locally. 
According to this one might wonder wether the missing dust attenuation comes from higher initial cmb temp than expected

**Comment by HanDeBruijn** [comment]
Instead of starting with reading the paper on top, I think it's better to absorb at first the one titled [_Light Speed Expanding Hubble-Hawking Universe_](https://www.researchgate.net/publication/369204981_Light_Speed_Expanding_Hubble-Hawking_Universe). Main ingredients of the author's theory are the following.
- The _currently believed Hubble radius_ of the (observable) universe is , $R_0 = (c/H_0)$ , where $c=$ speed of light, $H_0=$ Hubble parameter.
- _Currently believed cosmic critical density is_ , $\rho_0 = (3 H_0^2)/(8\pi G)$ , i.e. the well-known &Lambda;CDM critical mass density, where $G=$ gravitational constant.
_Considering the product of currently believed cosmic critical density and Hubble volume_ gives the total mass $M_0$ of the (observable) universe.

$$
M_0 = V_0\rho_0 = \left(\frac{4\pi}{3}\right)\left(\frac{c}{H_0}\right)^3.\frac{3 H_0^2}{8\pi G}= \frac{c^3}{2 G H_0} \quad \Longrightarrow \quad \frac{2 G M_0}{c^2} = \frac{c}{H_0} = R_0
$$

_On re-arranging this mass expression,_ it _clearly indicates something new about the current universe in terms of current **cosmic black hole**_ [ .. ] as has been motivated at ACG with [postings](https://github.com/orgs/a-cosmology-group/discussions/256) by @HanDeBruijn and @ExpEarth.

But then physically unmotivated **mathematism** (@budrap00) sneaks in.
_Currently believed cosmic temperature_ $\Theta_{CMB}$ _seems to be equal to the [**geometric mean**](https://en.wikipedia.org/wiki/Geometric_mean) of Hawking temperature of_ [_Planck mass_](https://en.wikipedia.org/wiki/Planck_units), $m_P$ [ .. ] _and [Hawking temperature](https://en.wikipedia.org/wiki/Hawking_radiation) of current cosmic Hubble mass_ $M_0$ , where $\hbar=$ reduced [Plank constant](https://en.wikipedia.org/wiki/Planck_constant) and $k_B=$ [Boltzmann constant](https://en.wikipedia.org/wiki/Boltzmann_constant) in

$$
\Theta_{CMB} = \frac{\hbar c^3}{8\pi k_B G \sqrt{m_p M_0}} \quad \mbox{with} \quad m_P = \sqrt{\frac{\hbar c}{G}}
$$

Yes of course, and then (approximately) consistent numbers come out for the Hubble parameter $H_0$ and the CMB temperature $\Theta_{CMB}$. But there are a myriad other ways to arrive at these results, as exemplified in [writeups](https://github.com/orgs/a-cosmology-group/discussions/276#discussioncomment-10923246) by @PatrGT and @sahil5d.

So far for the current paper. Let's go now to the [_Wrong Definition and Wrong Implications of Cosmic Red Shift_](https://www.onlinescientificresearch.com/articles/wrong-definition-and-wrong-implications-of-cosmic-red-shift-correction-and-possible-solutions.pdf).

Essential is a quote from page 1 in the paper. _If it is believed that, known physical laws of atomic and nuclear physics are applicable to other galaxies, then one can assume that, energy of photon at any galaxy is same as energy of photon coming from a laboratory resting in Milky Way._ That belief is not shared by people (like me) who adhere to the Variable Mass Theory. But okay, putting this issue aside, let us take a look at formula (3). Exactly the same one is displayed in a thread by @mikehelland: [_Time Dilated Past hypothesis_](https://github.com/orgs/a-cosmology-group/discussions/160). With analogous consequences.

$$
d_G = \left(\frac{z}{1+z}\right)\left(\frac{c}{H_0}\right) \quad \mbox{vs.} \quad d = -c\tau = \frac{c}{H}.\frac{z}{1+z}
$$

**Conclusion.** The "new" cosmic redshift $z_{new}$ is not really new for ACG members. It has been present [in diguise](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7645966) for quite some time here.

**Edit.** Following the math in the paper a few more formulas for those who care.

$$
M_0 = \frac{c^3}{2 G H_0} \quad \Longrightarrow \quad H_0 = \frac{c^3}{2 G M_0}
$$

Analogously they define sort of Hubble parameter $\eta_P$ for Planck mass:

$$
\eta_P = \frac{c^3}{2 G m_P} = \frac{c^3}{2 G \sqrt{\hbar c / G}} = \frac{1}{2}\sqrt{\frac{c^5}{\hbar G}}
$$

Calculating the (standard) Hubble parameter from standard CMB temperature:

$$
\Theta_{CMB} = \frac{\hbar c^3}{8\pi k_B G \sqrt{M_0 m_P}} \quad \Longrightarrow \quad \left(\frac{4\pi k_B \Theta_{CMB}}{\hbar}\right)^2 = \frac{c^3}{2 G M_0}.\frac{c^3}{2 G m_P} = H_0.\eta_P
$$

$$
H_0 = \frac{1}{\eta_P} \left(\frac{4\pi k_B \Theta_{CMB}}{\hbar}\right)^2 = 2\sqrt{\frac{\hbar G}{c^5}} \left(\frac{4\pi k_B \Theta_{CMB}}{\hbar}\right)^2
$$

Giving a CMB temperature = 2.726 K corresponding with a Hubble parameter $H_0=$ 67 km/s/Mpc.

**Reply by LifeIsStrange** [reply]
Amazing writeup, you are indeed very talented, sadly I cannot yet fully understand the semantics behind all the equations but you are being very helpful!
Also very curious on the evidence for blackhole cosmology.

https://github.com/orgs/a-cosmology-group/discussions/276#discussioncomment-10923246
> It is noticed that the large numbers coincidence is one equation with six (un)knowns. This means that only one (un)known can be calculated from the others. All the rest must be deemed as circular reasoning.

new paper from the same group I mentionned today (but other group from your original answer) mention that most previous issues with circular logic with unknown physical constants have been recently solved
> https://link.springer.com/article/10.1007/s10773-024-05570-6#:~:text=unsolvable%20circular%20argument

section 4
> However, in 1987, Cohen [[35](https://link.springer.com/article/10.1007/s10773-024-05570-6#ref-CR35)], who did a similar derivation, also pointed out that this would lead to an unsolvable circular argument, as there was no known method at the time to find the Planck units independently of calculating them from G, 
, or c. As recently as 2016, in an interesting paper by McCulloch [[36](https://link.springer.com/article/10.1007/s10773-024-05570-6#ref-CR36)], he highlighted the circular problem. In 2017, Haug [[37](https://link.springer.com/article/10.1007/s10773-024-05570-6#ref-CR37)] was the first to publish a method for finding the Planck length independently of G, and multiple publications on this topic have appeared since then, see for example [[28](https://link.springer.com/article/10.1007/s10773-024-05570-6#ref-CR28)].

the speed of gravity, plnack length, h0 and cmbtemp can all be mathematically determined without circularity, thanks to recent mathematical advances

**Reply by LifeIsStrange** [reply]
Also the gravitational constant is not intrinsically a primitive since it can be defined as a composite via the planck lenght, this proves that the universe is quantized/discrete
https://www.mdpi.com/2218-1997/8/9/454

**Reply by HanDeBruijn** [reply]
@LifeIsStrange . See my reply [elsewhere](https://github.com/orgs/a-cosmology-group/discussions/277#discussioncomment-11893004). I have nothing to add.

**Comment by RedshiftDrift** [comment]
The paper "Wrong Definition and Wrong Implications of Cosmic Red Shift (Correction and Possible Solutions)" is wrong from the start, so I have not read it past the first page.

  One cannot obtain a new physical understanding by changing definitions!  It is like trying to make my house ten times larger than 120 square-meters by using the "correct definition of length", which is of course the 'foot'.  Then my house is 1200 square-feet which gives me room for more furniture!  This is as ridiculous as what they do in the paper.

  There are thousands of possible wrong ideas, publishing one that coincidentally gets the CMB temperature is a waste of everybody's time.
